packages/core/src/machines/actors/auth/signIn.ts

```diff
@@ -0,0 +1,276 @@
+import { createMachine, sendParent, assign, sendUpdate } from 'xstate';
+import { get } from 'lodash';
+
+import { AuthEvent, AuthChallengeNames, SignInContext } from '../../../types';
+import { Auth } from 'aws-amplify';
+
+export const signInActor = createMachine<SignInContext, AuthEvent>(
+  {
+    initial: 'init',
+    id: 'signInActor',
+    states: {
+      init: {
+        always: [
+          { target: 'autoSignIn', cond: 'shouldAutoSignIn' },
+          { target: 'signIn' },
+        ],
+      },
+      autoSignIn: {
+        invoke: {
+          src: 'signIn',
+          onDone: { actions: 'setUser', target: '#signInActor.resolved' },
+          onError: '#signInActor.signIn',
+        },
+      },
+      signIn: {
+        initial: 'edit',
+        exit: 'clearFormValues',
```

Comment:
Does this work? Anytime I switch between Sign in and Sign Up I get this error.

`Warning: No implementation found for action type 'clearFormValues'`

Reply:
Yes it works, but I found this behavior too. I think the problems are ran in arbitrary order, so 'clearFormValues' is removed before `signIn` machine exits out. Let me see if tweaking this to "resolved" state fixes this.